tests/libcxxthrd/enc/enc.cpp

```diff
@@ -57,12 +57,15 @@ extern "C" int close(int fd)
 
 static std::vector<std::function<void*()>> _thread_functions;
 static int _next_enc_thread_id = 0;
-int enc_key = 0;
+int enc_key = 0; // Monotically increasing enclave key
```

Comment:
NTH: Make these counters use std::atomics; you can get thread-safe increment for free.

Reply:
Not implementing this now.